packages/database/scripts/lint_functions.ts

```diff
@@ -0,0 +1,50 @@
+import { exec, spawnSync } from "node:child_process";
```

Comment:
let's try to stick with camelCase to be consistent with the other files in this folder